web/packages/teleterm/src/ui/services/usage/usageService.ts

```diff
@@ -179,7 +180,7 @@ export class UsageService {
       'usageReporting.enabled'
     ).value;
 
-    if (!isCollectingUsageMetricsEnabled) {
+    if (!(ENV.prehogAddress && isCollectingUsageMetricsEnabled)) {
```

Comment:
I feel like it's better to spell out conditions like these explicitly instead of having to apply De Morgan's laws each time when reading this. 😏

```suggestion
    if (!ENV.prehogAddress || !isCollectingUsageMetricsEnabled) {
```

Reply:
Done :D